packages/react-inbox/src/components/Messages/styled.ts

```diff
@@ -1,14 +1,29 @@
 import styled from "styled-components";
 import deepExtend from "deep-extend";
 
+export const Container = styled.div(({ theme }) =>
+  deepExtend(
+    {
+      background:
+        theme.brand.inapp?.widgetBackground?.topColor &&
```

Comment:
maybe we should do optional chaining on brand too.

Reply:
yes probably.  good catch